ext/ldap/ldap.stub.php

```diff
@@ -0,0 +1,368 @@
+<?php
+
+//TODO: missing arginfo functions defined in C using
+
+#ifdef HAVE_ORALDAP
+/** @return resource|false */
+function ldap_connect(string $hostname = null, int $port = 389, string $wallet = null, string $wallet_passwd = null, int $authmode = GSLC_SSL_NO_AUTH) {}
+#else
+/** @return resource|false */
+function ldap_connect(string $hostname = null, int $port = 389) {}
+#endif
```

Comment:
This is extremely fishy, but an existing issue, so let's leave it like this for now...

However, the string arguments are not properly nullable, so they should be `= UNKNOWN`.